test/test_secret_manager_with_cli.py

```diff
@@ -30,6 +30,9 @@ def test_secret_manager(base_url, headers):
 
     createSecret.createSecret(base_url, "secretData2", "secret002", None, "mysecret", "20d")
 
+    totalCount_version = listSecretVersionIds.listSecretVersionIds(base_url, "secret001")
```

Comment:
Could test a secret with 2 versions?

Reply:
Current, just one version in db, if you want test 2 version, you need merge putSecret function

Reply:
OK.